src/storage/index.ts

```diff
@@ -106,22 +116,21 @@ export class StorageDataManager {
 
     for (const [eventName, eventsData] of [...parsedEvents.entries()]) {
       switch (eventName) {
-        case EventName.SpaceCreated:
```

Comment:
also, can you explain a bit why this event handling is removed?

Reply:
Because it's deprecated logic related with supporting `Space.handle` filed and now we don't need make any storage calls on `SpaceCreated/SpaceUpdated` events. All logic has been moved to `UserNameRegistered/UserNameUpdated` (technically `DomainRegistered/DomainMetaUpdated`) events. 
Storage section in the squid should be refactored as it was designed with a bit different ideas. But will be better to do this after refactoring Domains pallet in blockchain.

Reply:
got it, so its not the whole event handling that's deleted, only storage calls that was needed in previous implementation regarding handle right?